README.md

```
@@ -1,23 +1,283 @@
-## 💬 커밋 타입 가이드
+# <img src="https://github.com/user-attachments/assets/937754a3-264a-4590-8dd5-f061965b2024" width='30'> 일하고 싶은 중장년 여성을 위한, 실행 공유형 구직 서비스
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)